### PR TITLE
Calling out support for Single-Region, Multi-Region, Multi-AZ for EKS

### PR DESCRIPTION
**Pull request by amitmavgupta**

### CODE REVIEW

File: Documentation/installation/requirements-eks.rst [hunk 1]
```diff
@@ -13,8 +13,10 @@ For more information on AWS ENI mode, see :ref:`ipam_eni`.
 
 .. tip::
 
-   If you want to chain Cilium on top of the AWS CNI, refer to the guide
-   :ref:`chaining_aws_cni`.
+   - If you want to chain Cilium on top of the AWS CNI, refer to the guide :ref:`chaining_aws_cni`.
+
+   - Cilium can also be brought up in a Single-Region, Multi-Region, Multi-AZ environment for EKS.
```
Comment (zacharysarah) [inline Documentation/installation/requirements-eks.rst]: Edits for clarity, use active voice, address the user as "you"
```suggestion
   - To chain Cilium on top of the AWS CNI, see :ref:`chaining_aws_cni`.

   - You can also bring up Cilium in a Single-Region, Multi-Region, or Multi-AZ environment for EKS.
```

**Review by zacharysarah — CHANGES_REQUESTED** [review]
@amitmavgupta Some small formatting changes required, otherwise LGTM.

Be sure to add a release note.

**Review by christarazi — COMMENTED** [review]
Thanks for the PR.

Could you squash all of the commits into one commit? Also, please sign-off the commit by doing `git commit --amend --signoff` and if you'd like, reword the commit to say "Call out support for ..." so that it's a bit shorter.

File: Documentation/installation/requirements-eks.rst [hunk 2]
```diff
@@ -13,8 +13,10 @@ For more information on AWS ENI mode, see :ref:`ipam_eni`.
 
 .. tip::
 
-   If you want to chain Cilium on top of the AWS CNI, refer to the guide
-   :ref:`chaining_aws_cni`.
+   - To chain Cilium on top of the AWS CNI, see :ref:`chaining_aws_cni`.
+
+   - You can also bring up Cilium in a Single-Region, Multi-Region, or Multi-AZ environment for EKS.
```
Comment (zacharysarah) [inline Documentation/installation/requirements-eks.rst]: No hyphens needed.
```suggestion
   To chain Cilium on top of the AWS CNI, see :ref:`chaining_aws_cni`.

   You can also bring up Cilium in a Single-Region, Multi-Region, or Multi-AZ environment for EKS.
```

**Review by zacharysarah — CHANGES_REQUESTED** [review]

**Review by zacharysarah — APPROVED** [review]
LGTM for docs

**Comment by christarazi** [comment]
I am seeing two commits that are meant to be the same, could you please squash the two commits into one?

**Pull request merged**